pages/glossary/keyboard-actionable-element.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Keyboard Actionable Element
+key: keyboard-actionable-element
+unambiguous: true
+objective: true
+input_aspects:
+  - Accessibility tree
+  - CSS styling
+  - DOM tree
+---
+
+An HTML element is _keyboard actionable_ if all the following are true:
+
+- the element is part of [sequential focus navigation][]; and
+- the element is [visible][] when [focused][]; and
+- the element can be [activated][] by use of keyboard.
+
+When it is not [focused][], an actionable element might be visible or not, this is not important as long at the element is [visible][] when it is [focused][]. Similarly, it doesn't matter whether or not the element can be [activated][] by mouse or other input device as long as it can be [activated][] by keyboard.
```

Comment:
```suggestion
When it is not [focused][], an actionable element might be visible or not, this is not important as long as the element is [visible][] when it is [focused][]. Similarly, it doesn't matter whether or not the element can be [activated][] by mouse or other input device as long as it can be [activated][] by keyboard.
```